### DISCUSSION TITLE
I COULDN'T  DISABLE GITHUB 2FA THERE IS NO BUTTON TO RECONFIGURE !

### DISCUSSION THREAD

**Discussion by abdimk** [community/community#69489] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

as i said in the title i could not disable the github 2fa system i want to disable it
https://ibb.co/yqYZZm0 

PLEASE HELP IT HAS BEEN LIKE 8 MONTHS !

**Comment by MSatyam-Mishra** [comment]
2FA sometimes work and sometime it doesn't!

**Reply by abdimk** [reply, marked as answer]
Thanks for your response mate!

**Comment by ktauchathuranga** [comment]
i don't think that there is an option to remove 2FA,
it is a requirement to enable two-factor authentication (2FA) for certain contributors on GitHub.com.
it's in your screenshot too,
![image](https://github.com/community/community/assets/79928743/fcd724a9-6166-4ef3-9794-09417f4091e3)

you can use an Authenticating app like [Authy](https://authy.com/), which makes it very easy, instead of SMS msg

**Reply by abdimk** [reply]
Why for certain contributors ?
